pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -183,11 +186,17 @@ static PyObject *get_values(PyObject *obj) {
         //  without going through and object array of Timestamps.
         if (PyObject_HasAttrString(obj, "tz")) {
             PyObject *tz = PyObject_GetAttrString(obj, "tz");
+            if (tz == NULL) {
+                return NULL;
+            }
             if (tz != Py_None) {
                 // Go through object array if we have dt64tz, since tz info will
                 // be lost if values is used directly.
                 Py_DECREF(tz);
                 values = PyObject_CallMethod(obj, "__array__", NULL);
+                if (values == NULL) {
```

Comment:
I'm +/-0 on having to do this here. If `values` is NULL the `return values` naturally propagates up the caller, who is also responsible for checking. But if it helps enforce patterns then this is harmless